internal/pkg/template/templates/environment/partials/ar-vpc-connector.yml

```diff
@@ -0,0 +1,48 @@
+AppRunnerVpcEndpointSecurityGroup:
+  Metadata:
+    'aws:copilot:description': 'A security group to services in this environment talk to App Runner private services'
```

Comment:
Is the sentence structure a bit odd here? (also nit: can we stay within 80 chars)

```
A security group for App Runner private services
```